lib/livebook/utils.ex

```diff
@@ -833,4 +833,20 @@ defmodule Livebook.Utils do
       []
     end
   end
+
+  @doc """
+  Returns logger metadata for a list of users or their identity data.
+  """
+  @spec logger_users_metadata(list(Livebook.Users.User.t() | map())) :: String.t()
+  def logger_users_metadata(datas) when is_list(datas) do
+    list =
+      for data <- datas do
+        for key <- [:id, :name, :email],
+            value = Map.get(data, key),
+            do: {key, value},
+            into: %{}
+      end
+
+    [users: inspect(list)]
```

Comment:
Doing the inspection here is problematic because if we have JSON logging in the future, then it will be sent as a string instead of a lists of objects in JSON. Maybe we can cross this bridge later but we may need a custom data structure here, so we can implement both JSON and String.Chars protocols.